### ISSUE TITLE
Compressed numbers don't get converted to Sympy 

### ISSUE THREAD

**Issue by Lyxal** [Vyxal/Vyxal#1854]
# UPDATE

The script now throws an error I guess we need to import sympy somewhere.

# Original
No clue how we missed that one.

# [Vyxal](https://github.com/Vyxal/Vyxal) `c`, 49 bytes 
```
»⟑]"Ȯ⁋æ≥ǔ}¼√ḋ¾d¯₴÷≤„~-↳§*≤4_⌐↳¯→ḟS∆Y↵Ḃ¥zjλ…°qẆ∷9»
```

[Try it Online!](https://vyxal.pythonanywhere.com/?v=2#WyJjIiwiIiwiwrvin5FdXCLIruKBi8Om4omlx5R9wrziiJrhuIvCvmTCr+KCtMO34omk4oCefi3ihrPCpyriiaQ0X+KMkOKGs8Kv4oaS4bifU+KIhlnihrXhuILCpXpqzrvigKbCsHHhuobiiLc5wrsiLCIiLCIiXQ==)

Gives 

```
stack.append(2190103378294521845700248699309885665950123333195645737170955981267389324534409006970256858120588676151695966441) 
```

Notice the lack of Sympy wrapper.

**Comment by PlaceReporter99** [comment]
Attempting a fix in #1855

**Comment by PlaceReporter99** [comment]
It now errors, probably because we do need to import something.
